.github/workflows/report-failure.yaml

```diff
@@ -0,0 +1,87 @@
+# This workflow is intended to be called by workflows in our various software
+# repos to have shared workflow failure reporting in the form of GitHub issues.
+on:
+  workflow_call:
+    inputs:
+      failure:
+        description: Whether to create an issue indicating a failed workflow run, or keep an existing issue open.
+        type: boolean
+        required: true
+      success:
+        description: Whether to close any open issues created by this workflow.
+        type: boolean
+        required: true
```

Comment:
Taking both `failure` and `success` is weird, especially when the action taken on them (below) is intended to be mutually-exclusive (e.g. `assert failure xor success`). Why can't this be a single boolean?